ironmon_tracker/Drawing.lua

```diff
@@ -30,7 +30,7 @@ function Drawing.drawBackgroundAndMargins(x, y, width, height)
 end
 
 function Drawing.drawPokemonIcon(pokemonID, x, y)
-	if not PokemonData.isValid(pokemonID) then
+	if not PokemonData.isImageIDValid(pokemonID) then
```

Comment:
Draw the icon as long as we have a valid image ID for it (aka include Eggs, Ghosts)